src/menu.cpp

```diff
@@ -312,19 +317,27 @@ void Menu::setTitle(std::string text)
   updatePosition();
 }
 
+void Menu::setLineCount(uint16_t rows)
+{
+  content->body.setLineCount(rows);
+  updatePosition();
+}
+
 void Menu::addLine(const std::string &text, std::function<void()> onPress,
-                   std::function<bool()> isChecked)
+                   std::function<bool()> isChecked, 
+                   bool updatePos)
 {
   content->body.addLine(text, std::move(onPress), std::move(isChecked));
-  updatePosition();
+  if(updatePos) updatePosition();
```

Comment:
Can you please put those into a new line? this kind of code formating is OK if you have multiple one line ifs together, but written like that it is dangerous (ask me how I know...)

Reply:
Ok, I'll do :)

Reply:
I could have lived without the brackets. but they donÄt bother me much. So for me this is OK like you did it now.

Reply:
Braces are good... it also tells the auto-formatter to not put short lines on one line ;)